README.md

```diff
@@ -30,6 +30,17 @@ requests need to be manually reviewed and merged by a human.
 It is ***highly recommended*** to also use [TagBot][tagbot], which automatically tags a release in your
 repository after the new release of your package is merged into the registry.
 
+### Requirements for packages in the General registry
+
+Packages registered in the General registry must be made in good faith.
+
+We do not accept packages or package registrations that:
+1. Are intentionally broken
```

Comment:
Not sure what this means?

Reply:
🤷

Reply:
Direct quote from Slack

Reply:
I guess Seth's IsTwo.jl is an example

Reply:
How about "obviously incorrect"?

Reply:
I'll change it to "obviously and intentionally incorrect." Since this is documentation about intent.

Reply:
The problem was not that it was broken, it was that you clearly didn’t even mean for it to work. People will register packages that don’t work, we’re not going to be able to stop that.

Reply:
I think "obviously and intentionally incorrect" covers that, right? It covers the case of authors not intending for their code to work correctly.

Reply:
How do we determine intent? That is, if I submit
```
"""
    istwo(x)

Return `true` if x is equal to two
"""
istwo(x::Number) = isapprox(x, 2)
```
does that pass muster?

Reply:
Basically, it's at the discretion of the registry maintainers 🤷 

The main reason for documenting this is that now when I close a bad-faith PR, I can use these rules as justification.

Reply:
If it's at the discretion of the registry maintainers, then *put that in there*.

Reply:
@StefanKarpinski has said in Slack that it is "absolutely at the discretion of the maintainers". I would suggest that we can scrap everything other reason and just use that.